charts/airbyte/templates/env-configmap.yaml

```diff
@@ -17,8 +17,8 @@ data:
   DATABASE_USER: {{ include "airbyte.database.user" . }}
   DB_DOCKER_MOUNT: airbyte_db
   FULLSTORY: {{ ternary "enabled" "disabled" .Values.webapp.fullstory.enabled }}
-  GCS_LOG_BUCKET: ""
-  GOOGLE_APPLICATION_CREDENTIALS: ""
+  GCS_LOG_BUCKET: {{ .Values.logs.gcs.bucket | default "" | quote }}
+  GOOGLE_APPLICATION_CREDENTIALS: {{ .Values.logs.gcs.credentials | default "" | quote }}
```

Comment:
I think you can remove the `default ""` bit.

See https://github.com/airbytehq/airbyte/pull/8736/files#diff-6f2647f2aed4a58e6f608dc5f977d94e39e237dd313adc6de33e66b8ced6482dR32

Reply:
@Nmaxime